packages/xo-web/src/common/intl/messages.js

```diff
@@ -2391,6 +2397,15 @@ const messages = {
   auditInactiveUserActionsRecord: 'User actions recording is currently inactive',
 
   // Licenses
+  allHostsMustBeBound: 'All hosts must be bound by a license',
+  bindXcpngLicenses: 'Bind XCP-ng licenses',
+  confirmBindingOnUnsupportedHost:
+    'You are about to bind professional support license{nLicense, plural, one {} other {s}} on older and unsupported XCP-ng version{nLicense, plural, one {} other {s}}. Are you sure do you want to do this?',
```

Comment:
```suggestion
  confirmBindingOnUnsupportedHost:
    'You are about to bind professional support license{nLicense, plural, one {} other {s}} on older and unsupported XCP-ng version{nLicense, plural, one {} other {s}}. Are you sure you want to continue?',
```
What is the consequence of doing this?